plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/Session.kt

```diff
@@ -152,6 +152,9 @@ class Session(val tabID: String, val project: Project) {
             }
         }
 
+    val conversationIdUnsafe: String?
```

Comment:
Why is this named `conversationIdUnsafe`? Because it can be null, too?

Reply:
Yep